platforms/hyperledger-besu/charts/node_orion/templates/deployment.yaml

```diff
@@ -197,6 +203,10 @@ spec:
             cat {{ .Values.node.mountPath }}/genesis/genesis.json.base64 | base64 -d > $BESU_GENESIS_DIR/genesis.json
             tr -d "\n\r" < /secrets/orion_key.pub > /secrets/nodekey.pub
             args="--rpc-http-enabled --rpc-http-api=ETH,NET,IBFT,DEBUG,ADMIN,WEB3,EEA,PRIV --rpc-ws-enabled --rpc-ws-api=ETH,NET,WEB3 --p2p-port {{ .Values.node.ports.p2p }} --rpc-http-port {{ .Values.node.ports.rpc }} --rpc-ws-port={{ .Values.node.ports.ws }}"
+            if [ {{ $.Values.orion.tls }} == "true" ]
+            then
+              tls_args="--privacy-tls-enabled --privacy-tls-keystore-file=/secrets/keystore.pkcs12 --privacy-tls-keystore-password-file=/secrets/keystore_password --privacy-tls-known-enclave-file=/secrets/known_server"
```

Comment:
Where are the changes to orion.conf to accept tls connections from besu clients? Did you try deploying the private smart contract after these changes?

Reply:
In the docs, the orion server configuration link is redirecting to the besu tls page, which is there in our code. I tried to deply the smart contract but it is still failing, looking into it.